### PR TITLE
Feature/workflows

### PR DESCRIPTION
**Pull request by QGavoille**

### CODE REVIEW

File: src/workflow/workflow.ts [hunk 1]
```diff
@@ -1,31 +1,31 @@
-import { IPkName, IRepositoryWritable } from "../repository/repository";
+import { IPkName, IRepositoryWritable } from "../repository/repository"
 
 export interface Context {
```
Comment (harpsdev) [inline src/workflow/workflow.ts]: Rename to WorkflowContext

File: src/workflow/workflow.ts [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
   source: string;
-  data: any;
+  data: {record: any, additionnal?: any};
```
Comment (harpsdev) [inline src/workflow/workflow.ts]: rename data to ActionContext

**Review by harpsdev — CHANGES_REQUESTED** [review]

File: src/workflow/workflow.ts (hunk 2 above)
Comment (QGavoille) [reply src/workflow/workflow.ts]: fixed by [9dec683](https://github.com/all41-dev/server/pull/37/commits/9dec6838e68d41361af6eb59aa00d23023ba2105)

File: src/workflow/workflow.ts (hunk 1 above)
Comment (QGavoille) [reply src/workflow/workflow.ts]: fixed by [9dec683](https://github.com/all41-dev/server/pull/37/commits/9dec6838e68d41361af6eb59aa00d23023ba2105)

**Pull request merged**